packages/axios-extension/src/abap/adt-catalog/services/transportrequest-service.ts

```diff
@@ -26,24 +23,34 @@ export class TransportRequestService extends AdtService {
     }
 
     /**
+     * TransportRequestService API function to create a new transport number.
      *
+     * @param packageName Package name for deployment
+     * @param appName Fiori project name for deployment. A new project that has not been deployed before is also allowed
      * @param description Description of the new transport request to be created
      * @returns Newly created transport request number
      */
-    public async createTransportRequest(description: string): Promise<string> {
+    public async createTransportRequest(packageName: string, appName: string, description: string): Promise<string> {
```

Comment:
With this PR we no longer support any ABAP object but very specifically support only UI5 apps?

Reply:
Thanks @theStiggySpd for the clarification here: https://github.com/SAP/open-ux-tools/pull/793#issuecomment-1309224867

Resolving